hadoop-ozone/dist/src/main/compose/common/ec-test.sh

```diff
@@ -18,7 +18,7 @@
 start_docker_env 5
 
 ## Exclude virtual-host tests. This is tested separately as it requires additional config.
-execute_robot_test scm -v BUCKET:erasure --exclude virtual-host s3
+execute_robot_test scm -v BUCKET:erasure --exclude virtual-host --exclude ec-storage-class s3
```

Comment:
I would like to avoid having to `--exclude` yet another test in several scripts.  We can add the new robot test file in `ec/` instead of `s3/`.